integration-tests/tests/postgres.test.ts

```diff
@@ -82,13 +89,13 @@ describe('postgres', async () => {
     ])
 
     await pg.query(
-      'INSERT INTO encrypted (encrypted_text) VALUES ($1), ($2), ($3)',
+      'INSERT INTO encrypted (encrypted_text) VALUES ($1::jsonb), ($2::jsonb), ($3::jsonb)',
       ciphertexts,
     )
 
     const res = await pg.query(`
-      SELECT encrypted_text FROM encrypted
-      ORDER BY cs_ore_64_8_v1(encrypted_text)
+      SELECT encrypted_text::jsonb FROM encrypted
+      ORDER BY eql_v2.order_by(encrypted_text) ASC
```

Comment:
Heads up that `eql_v2.order_by` is still needed for ordering @calvinbrewer. Tried not using the function while pairing with @tobyhede but it didn't work as intended.